### ISSUE TITLE
[Feature request] I should be able to click link in another other app to open a specific audio anchor file

### ISSUE THREAD

**Issue by mstyp** [flackbash/AudioAnchor#172]
TLDR: Its to difficult to get to files in audio anchor from other apps or from home screen. In an ideal world let me open audio files in anchor via some intent / linking mechanism like `[link to meeting recording](content://file/path/file.mp3)` or with a "share link" button, so I can jump right into listening to the file from my task manager.

Audio Anchor is perfect for work related audio tasks when it comes to actually listening to the audio. But its just to hard to actually get to that point. Heck there are not even any shortcuts I can add to a home-screen (that would not help me when I am feeling peak lazy but it would help sometimes when I am just feeling lazy)

But because there is no good way that I can see to quickly link to audio anchor files, you get this weird situation where its easier for me to do tasks like "read this academic paper" then it is to do things like "listen to this fun podcast episode that also has work related information", because (during a time when I was not feeling super lazy) I setup a tasker workflow that looks at this task list, picks one at random, and opens the relevant task. and if its a "read this academic paper" task all I need to do then is click a markdown link and start reading. it takes literally less then 2 seconds and once I've started its easy to keep going. 

But whenever the task is something along the lines of "listen to X", I always just hit the back button and try again for something simpler, because I am feeling to lazy to actually open the anchor app and search for the relevant file. that requires to much thinking when I am in peak lazy mode. 

I used to use VLC for this since I could use markdown like `[link to meeting recording](content://file/path/file.mp3)` to open the file with VLC, but I use VLC watch tv and listen to music. its my "fun" app, and its not good to mix fun and work. I occasionally found myself pausing to look for more fun videos to watch. Plus, VLC also does not have a quick "skip forward / back 10 seconds" or a "bookmarks" feature, both of which are necessary for when listening to meeting recordings or public hearings or press conferences or other audio that has boring sections that can be skipped through or poor quality audio.

**Comment by flackbash** [comment]
I get your point. I think the main problem with this is, that AudioAnchor is directory-based, and not file-based.

For storing the current playback position, bookmarks, etc., AudioAnchor needs to add the parent directory (or the parent directory of the parent directory) in which an audio file is stored to its library. That means, if I implemented the functionality of being able to open a file link with AudioAnchor, this action would have to automatically add the whole parent directory to the library (or the parent directory of the parent directory).
So it would not only add this specific file to the library, but all the audio files in the given directory. Which is probably not what you expect when you perform an action on just a single file. Additionally, AudioAnchor would have to make the implicit decision at this point of whether to add just the parent directory or the parent of the parent directory to the library (which is something the user usually can decide themselves when adding a directory).

**Comment by mstyp** [comment]
Thats really interesting, i can see why you dont allow opening random audio files from standard file browser then.  But for me I dont really need the ability to open random audio files, I only want to get at files I already know are in anchor quicker. Its hard enough to do that when you have hundreds of audio files and you have to scroll round for what feels like forever, add to that the fact that 75% of them have names like `2022-04-13_14:42.mp3` or `media(13).mp3`, plus me generally doing this when I am feeling lazy, and its not a great recipe for getting things done. last two issues are pretty specific to me but I cant be the only one with foulders with hundreds of audio files and a desire to quickly get to the right one.

I dont know enough about how audio anchor works (or about how android works), but i think this i called "deep linking"? is that the same thing as opening a file link? idk. https://developer.android.com/training/app-links/deep-linking

Appreciate all the work you do! This app is fantastic.
